vault/token_store.go

```diff
@@ -310,7 +310,8 @@ func (ts *TokenStore) paths() []*framework.Path {
 			Fields: map[string]*framework.FieldSchema{
 				"token": {
 					Type:        framework.TypeString,
-					Description: "Token to lookup (POST request body)",
+					Description: "Token to lookup",
+					Query:       true,
```

Comment:
Note for reviewer: The change to the description is because it is entirely possible to call this API like:
```
vault read auth/token/lookup token=...
```
and thus the qualifier `(POST request body)` is untrue.